frontend/src/pages/org/settings.ts

```diff
@@ -301,6 +309,44 @@ export class OrgSettings extends LiteElement {
     `;
   }
 
+  private renderBilling() {
+    return html`<div class="rounded border">
+      <form @submit=${this.onOrgBillingSubmit}>
+        <div class="grid grid-cols-5 gap-x-4 p-4">
+          <div class="col-span-5 md:col-span-3">
+            <sl-input
```

Comment:
Would be good to add:

`inputmode="numeric"` for good keyboard selection on mobile / touch input devices

`<span slot="suffix">Minutes</span>` for clarity / alignment with how we define values in the crawl workflows

Reply:
Implemented